<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartScreenSnapper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/image/icon.png</normaloff>:/image/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionPrint"/>
    <addaction name="actionCloseAllNotSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionOpenSource"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionCopy"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTool"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>工具条</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>33</width>
     <height>33</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionScreenSnap"/>
   <addaction name="actionActiveWindowSnap"/>
   <addaction name="actionCursorSnap"/>
   <addaction name="actionFreeSnap"/>
   <addaction name="actionGIF"/>
  </widget>
  <widget class="QDockWidget" name="dockDocumentList">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>298</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>文档列表</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QListWidget" name="listDocument"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存(&amp;S)...</string>
   </property>
   <property name="toolTip">
    <string>保存文档</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionScreenSnap">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/ScreenSnap.png</normaloff>:/image/ScreenSnap.png</iconset>
   </property>
   <property name="text">
    <string>全屏截图</string>
   </property>
   <property name="toolTip">
    <string>全屏截图</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于(&amp;A)...</string>
   </property>
  </action>
  <action name="actionActiveWindowSnap">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/ActiveWindow.png</normaloff>:/image/ActiveWindow.png</iconset>
   </property>
   <property name="text">
    <string>活动窗口截图</string>
   </property>
   <property name="toolTip">
    <string>活动窗口截图</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/Setting.svg</normaloff>:/image/Setting.svg</iconset>
   </property>
   <property name="text">
    <string>设置(&amp;S)...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionCursorSnap">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/CursorSnap.png</normaloff>:/image/CursorSnap.png</iconset>
   </property>
   <property name="text">
    <string>截取光标</string>
   </property>
   <property name="toolTip">
    <string>截取光标</string>
   </property>
  </action>
  <action name="actionCloseAllNotSave">
   <property name="text">
    <string>关闭所有文档且不保存...</string>
   </property>
  </action>
  <action name="actionOpenSource">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/GitHub.svg</normaloff>:/image/GitHub.svg</iconset>
   </property>
   <property name="text">
    <string>代码开源...</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionFreeSnap">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/FreeSnap.png</normaloff>:/image/FreeSnap.png</iconset>
   </property>
   <property name="text">
    <string>自由截图</string>
   </property>
   <property name="toolTip">
    <string>自由截图</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>检查更新...</string>
   </property>
   <property name="toolTip">
    <string>检查更新</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>打印(&amp;P)...</string>
   </property>
  </action>
  <action name="actionGIF">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/image/GIFRecord.png</normaloff>:/image/GIFRecord.png</iconset>
   </property>
   <property name="text">
    <string>录制GIF</string>
   </property>
   <property name="toolTip">
    <string>录制GIF</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
